### ISSUE TITLE
Errors and messages indicated in console only via color

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#7023]
### System details

    RStudio Edition : Desktop or Server
    RStudio Version : 1.4.424
    OS Version      : any
    R Version       : any

### Steps to reproduce the problem
- do something in the Console to generate messages (e.g. `message("foo")` or errors
- try to distinguish these from regular output if you are using a screen-reader or are color-blind

### Describe the problem in detail
- color is the only thing being used to mark this type of output

### Describe the behavior you expected
- should be another way to tell these lines are messages/errors not solely relying on color

Breaking this out from #2574

- [x] I have read the guide to submitting good bug reports at https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports . 
- [x] I have installed the latest version of RStudio and confirmed that the issue still persists.
- [x] If I am reporting a RStudio crash, I have included a diagnostics report. https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.
